mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/AttachmentMapperTest.java

```diff
@@ -159,7 +160,7 @@ void getRelatedMessageIdsShouldReturnMessageIdWhenStoredWithMessageId() throws E
         MessageId messageId = generateMessageId();
         AttachmentId attachmentId = attachmentMapper.storeAttachmentsForMessage(ImmutableList.of(ParsedAttachment.builder()
             .contentType("content")
-            .content("".getBytes(StandardCharsets.UTF_8))
+            .content(ByteSource.wrap("".getBytes(StandardCharsets.UTF_8)))
```

Comment:
it's a very complex way of providing an empty ByteSource

Reply:
But it works and is simple to understand / write.

You have an alternative?

Reply:
`ByteSource.empty()` ?

Reply:
Fair